packages/terra-clinical-detail-view/src/DetailView.jsx

```diff
@@ -155,14 +155,17 @@ const DetailView = (props) => {
     dividedDetails = details;
   }
 
+  const showTitleSection = (titleElement || secondaryTitlesElements.length || subtitleElements.length || accessoryElement);
   return (
     <section {...attributes} aria-labelledby={titleId}>
+      {showTitleSection && (
```

Comment:
Since `showTitleSection` is dependent upon several variables, we should add some jest unit tests to make sure all cases are covered.

Reply:
Added tests for all the sections : https://github.com/cerner/terra-clinical/commit/c598ecc9301aad24ee248a1a5a560192d371c448#diff-cbf723360a8e7bc0de6c6c28d8f6b3a65156666904f3b54b8a724cbe75bf37cf